src/config/dynamic-theme-fixes.config

```diff
@@ -18714,6 +18714,22 @@ IGNORE INLINE STYLE
 
 ================================
 
+terraria.wiki.gg
+
+CSS
+:root {
+    --darkreader-bg--theme-content-background: transparent;
+    --darkreader-bg--ooui--menuselect-background-color: #111111;
+}
+body, .navbox, #mw-head, #footer, #simpleSearch #searchButton, #simpleSearch #searchButton:hover, #searchInput:hover ~ #searchButton, #searchInput:focus ~ #searchButton, #mw-panel .portal, #mw-panel #p-logo + .portal {
+    background: none;
+}
```

Comment:
What does this fix? The backgrounds are not breaking anything for Dark Reader.

Reply:
See my comment below. This gets rid of the mid brown background colors and images. The mid brown has a value of 35% brightness. It's incredibly bright compared to the rest of the web. This also gets rid of the white background image.

Reply:
Removing them is not the solution in this situation, you are better of by using this CSS rule instead:

```
    filter: brightness(50%) sepia(40%) !important;
```

To reduce the brightness of the image.